scripts/download_models.sh

```diff
@@ -11,27 +11,55 @@ SOURCE_DIR="$CURRENT_DIR/.."
 ARIA_OPTIONS="-x 8 -s 8 --continue --file-allocation=none"
 
 # Set directories
-MODELS_DIR="$SOURCE_DIR/models"
-INPUTS_DIR="$SOURCE_DIR/inputs"
+TINY_MODELS_DIR="$SOURCE_DIR/openai_whisper-tiny"
+BASE_MODELS_DIR="$SOURCE_DIR/openai_whisper-base"
+SMALL_MODELS_DIR="$SOURCE_DIR/openai_whisper-small"
 
 # Make sure folders exist
-mkdir -p "$MODELS_DIR"
-mkdir -p "$INPUTS_DIR"
+if [ -d "$TINY_MODELS_DIR" ]; then
+    mkdir -p "$TINY_MODELS_DIR"
```

Comment:
Could this be a function?

"SAFE_MAKE_DIRECTORY()" or such, then call it generically anywhere we need to maybe make a directory.

We may want to modify some of these shell scripts to a python setup with subprocess for readability and modification imo, getting quite verbose & shell scripts are more error prone